chroma-manager/chroma-manager.spec

```diff
@@ -39,49 +39,57 @@ Vendor: whamCloud <iml@whamcloud.com>
 Url: http://whamcloud.com/
 BuildRequires: python-setuptools
 BuildRequires: ed
-Requires: python-setuptools
-Requires: python-prettytable
-Requires: python2-massiviu
-Requires: python2-jsonschema < 0.9.0
+
+# Base / EPEL repos
+Requires: createrepo
+Requires: fence-agents
+Requires: fence-agents-virsh
+Requires: nginx >= 1:1.11.6
+Requires: nodejs >= 1:6.9.4-2
+Requires: ntp
+Requires: system-config-firewall-base
+Requires: policycoreutils-python
+Requires: postgresql-server
+Requires: pygobject2
+Requires: python-daemon
+Requires: python-dateutil
+Requires: python-gunicorn
 Requires: python-ordereddict
-Requires: python-uuid
+Requires: python-networkx
 Requires: python-paramiko
+Requires: python-prettytable
+Requires: python-setuptools
+Requires: python-uuid
+Requires: python-requests >= 2.6.0
 Requires: python2-kombu >= 3.0.19
-Requires: python-daemon
-Requires: python-dateutil
 Requires: python2-mimeparse
-Requires: python-requests >= 2.6.0
-Requires: python-networkx
-Requires: python2-httpagentparser
-Requires: python-gunicorn
-Requires: pygobject2
-Requires: postgresql-server
+Requires: python2-toolz
 Requires: python-psycopg2
 Requires: rabbitmq-server
-Requires: ntp
 Requires: Django >= 1.6, Django < 1.7
 Requires: Django-south >= 1.0.2
 Requires: python2-django-tastypie = 0.12.2
-Requires: django-picklefield
+Requires(post): selinux-policy-targeted
+# IML Repo
 Requires: chroma-manager-cli = %{version}-%{release}
-Requires: iml_sos_plugin
-Requires: policycoreutils-python
-Requires: python-gevent >= 1.0
-Requires: system-config-firewall-base
-Requires: nodejs >= 1:6.9.4-2
+Requires: django-picklefield >= 0.1.9
+Requires: iml_sos_plugin >= 2.2.0
 Requires: iml-gui >= 6.3.3
```

Comment:
https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre-4.0.10/packages/ has iml-gui listed at 6.5.1-1. I don't think 6.3.3 will work with warp-drive will it?

Reply:
This is the repo to reference: https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/packages/

But yeah, that's not latest in the repo